src/python/WMCore/WorkQueue/Policy/Start/MonteCarlo.py

```diff
@@ -66,6 +67,9 @@ def split(self):
             remainingLumis = ceil(nEvents % self.args['SliceSize'] / self.args['SubSliceSize'])
             nLumis += remainingLumis
             jobs = ceil(nEvents/self.args['SliceSize'])
+            
+            if nLumis > int(os.environ.get('MaxLumisPerWQElement', 200000)):
```

Comment:
How about we call it `MAX_LUMIS_PER_WQE`? In addition to that, did you test the config changes that set this variable?